Source: http://www.slackware.com/~alien/slackbuilds/chromium/build/patches/chromium_widevine.patch
Upstream: Not upstreamable
Reason: Make Chromium support the Widevine Content Decryption Module plugin

Find the version of Widevine using this commandline on the chrome binary from the deb archive which is
downloaded by chromium-stable-widevine-plugin and/or chromium-stable-flash-plugin.
$ strings ./opt/google/chrome/chrome | grep -C 1 " (version:" | tail -1

--- chromium-39.0.2171.95/third_party/widevine/cdm/widevine_cdm_version.h.orig	2014-12-10 05:36:13.000000000 +0100
+++ chromium-39.0.2171.95/third_party/widevine/cdm/widevine_cdm_version.h	2014-12-28 20:47:39.080795240 +0100
@@ -14,4 +14,8 @@
 //  - WIDEVINE_CDM_VERSION_STRING (with the version of the CDM that's available
 //    as a string, e.g., "1.0.123.456").
 
+#include "third_party/widevine/cdm/widevine_cdm_common.h"
+#define WIDEVINE_CDM_AVAILABLE
+#define WIDEVINE_CDM_VERSION_STRING "1.4.6.738"
+
 #endif  // WIDEVINE_CDM_VERSION_H_
--- chromium-39.0.2171.95/third_party/widevine/cdm/widevine_cdm.gyp.orig	2014-12-10 05:54:35.000000000 +0100
+++ chromium-39.0.2171.95/third_party/widevine/cdm/widevine_cdm.gyp	2014-12-28 20:50:48.163269989 +0100
@@ -7,7 +7,7 @@
     'widevine_cdm_version_h_file%': 'widevine_cdm_version.h',
     'widevine_cdm_binary_files%': [],
     'conditions': [
-      [ 'branding == "Chrome"', {
+      [ 'branding == "Chromium"', {
         'conditions': [
           [ 'chromeos == 1', {
             'widevine_cdm_version_h_file%':
@@ -54,7 +54,7 @@
       'target_name': 'widevinecdmadapter',
       'type': 'none',
       'conditions': [
-        [ 'branding == "Chrome" and enable_pepper_cdms==1', {
+        [ 'branding == "Chromium" and enable_pepper_cdms==1', {
           'dependencies': [
             '<(DEPTH)/ppapi/ppapi.gyp:ppapi_cpp',
             '<(DEPTH)/media/media_cdm_adapter.gyp:cdmadapter',
